kubejs/server_scripts/base/featrues/chickenEggs.js

```diff
@@ -0,0 +1,31 @@
+// Change chicken egg generation from natural generation to being fed a seed
```

Comment:
Can you wrap the whole thing in an `if (feature(` that will make it so people can turn it off if they want and is self documenting. You can use the full comment as the feature name